src/App.js

```diff
@@ -1,43 +1,22 @@
 import "./App.css";
-import logo from "./logo.png";
+import {Routes, Route} from "react-router-dom";
+import {Home, LikedVideos, WatchLater, Login, Signup, VideoListing} from "../src/frontend/pages/index";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Routes>
+        <Route path="/" element={<Home />}/>
+        <Route path="/login" element={<Login />}/>
+        <Route path="/signup" element={<Signup />}/>
+        <Route path="/LV" element={<LikedVideos />}/>
+        <Route path="/WL" element={<WatchLater />}/>
```

Comment:
Suggestion: Instead of writing LV, WL, write `liked-videos`, `watch-later`. It's a good practice